task/push-build-file-oci-ta/0.1/push-build-file-oci-ta.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: "docker"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "image-build, appstudio"
+  name: push-build-file-oci-ta
```

Comment:
Could you add the `recipe.yaml` (every oci-ta task should have one) for this task, so that auto-generation works properly?

Reply:
Updated.